### ISSUE TITLE
Bug: Backlinks visible only when Graphview is enabled

### ISSUE THREAD

**Issue by adityatelange** [Fevol/starlight-site-graph#2]
When `Graph View` is disabled using below config, `backlinks` are not present on any page after you build it using `npm run build`

```diff
import starlight from '@astrojs/starlight'
import { defineConfig } from 'astro/config'
import starlightThemeObsidian from 'starlight-theme-obsidian'

export default defineConfig({
    integrations: [
        starlight({
            plugins: [starlightThemeObsidian({
++               graphConfig: {
++                    visibilityRules: []
++               }
            })],
            title: 'My Docs',
        }),
    ],
})
```

**Comment by adityatelange** [comment]
Not sure but PageGraph pulls it from `'sitegraph/sitemap.json` file but PageBacklinks doesn't


https://github.com/Fevol/starlight-site-graph/blob/2eb770c9f1b21274ca63e4fcb75553dda5c460a3/packages/starlight-site-graph/components/graph/PageGraph.astro#L68

https://github.com/Fevol/starlight-site-graph/blob/2eb770c9f1b21274ca63e4fcb75553dda5c460a3/packages/starlight-site-graph/components/backlinks/PageBacklinks.astro#L30

**Comment by Fevol** [comment]
Thank you very much for reporting the issue! 
I can't believe that I never noticed that the backlinks weren't being added on production websites (in `dev`, this works fine).
The issue was I had forgotten to remove an optimization step that wasn't being used anymore, which initially was meant to prevent the sitemap from being included in its entirety into the HTML (which is not desirable)
The issue has been resolved in [0.1.10](https://github.com/Fevol/starlight-site-graph/releases/tag/0.1.10)

**Comment by adityatelange** [comment]
Thank you for fixing this quickly, really appreciate it! Just built with the latest release and it works flawlessly. :+1:
